grafana-plugin/src/models/filters/filters.types.ts

```diff
@@ -0,0 +1,13 @@
+import { SelectOption } from 'state/types';
+
+export interface FiltersValues {
+  [key: string]: any;
+}
+
+export interface FilterOption {
+  name: string;
+  type: 'search' | 'options' | 'boolean' | 'daterange' | 'team_select';
```

Comment:
I think I saw `'search' | 'options' | 'boolean' | 'daterange' | 'team_select'` defined somewhere else as well, can this be extracted higher up to something like this, and reused?
```ts
export type FilterOptionType = 'search' | 'options' | 'boolean' | 'daterange' | 'team_select';
```